### ISSUE TITLE
Implement basic matomo metrics

### ISSUE THREAD

**Issue by BAAlexK** [bcgov/nr-rec-resources#163]
#### Description:
As a business, we need to see metrics on users, in order to better understand their usage, and gather insights that lead to more improvements of the website and increased satisfaction of users.

#### Acceptance Criteria:
- [ ] Setup basic/default metrics such as page views, visitors, time spent, referral domain/URL

Note: all custom events are covered in another ticket

**Development checklist**
- [ ] Implement tracking SDK
- [ ] Configure custom event tracking
- [ ] Set up data transformation rules
- [ ] Implement error handling

#### Dependencies
- Blocked by https://github.com/bcgov/nr-rec-resources/issues/434 and https://github.com/bcgov/nr-rec-resources/issues/259
- Blocking

**Comment by airliah** [comment]
we're good to go with clickstream :)

**Comment by airliah** [comment]
decision was made to go with Matomo due to issues with Clickstream
